addons/website_forum/views/website_forum.xml

```diff
@@ -441,30 +463,56 @@
     <t t-set="no_description_exist" t-value="bool(_forums.filtered(lambda f: not f.description))"/>
     <t t-set="sorted_forums" t-value="_forums.sorted(lambda f: (not f.description, f.sequence, f.id))"/>
 
+    <!-- Check for active options: the stored value may be used in sub-templates too  -->
+    <t t-set="opt_post_count" t-value="request.website.viewref('website_forum.opt_post_count').active"/>
+    <t t-set="opt_last_post" t-value="request.website.viewref('website_forum.opt_last_post').active"/>
+    <t t-set="opt_list_view" t-value="request.website.viewref('website_forum.opt_list_view').active"/>
+
     <!-- First, list all forums (those with descriptions first) -->
     <t t-foreach="sorted_forums" t-as="forum">
         <t t-set="has_desc" t-value="forum.description"/>
-        <div t-attf-class="mb-3 col-sm-6 #{not has_desc and 'col-md-3'}">
-            <div t-attf-class="#{has_desc and 'row' or 'd-flex flex-column h-100'} #{no_description_exist and has_desc and 'py-4 bg-200'}">
-                <div t-attf-class="#{has_desc and 'col-md-4 pr-md-0' or 'flex-lg-grow-1'}">
+        <div t-attf-class="mb-3 #{opt_list_view and 'col-sm-12' or 'col-sm-6'}">
+            <div class="row py-4 bg-200 mx-1 h-100 o_forum_row">
+                <div t-attf-class="o_forum_image_container pr-md-0 #{opt_list_view and 'col-md-3 o_forum_list_view' or 'col-md-4'}">
                     <a t-attf-href="/forum/#{slug(forum)}">
-                        <div t-attf-class="oe_img_bg rounded shadow-sm py-4 px-5 o_wforum_forum_card_bg flex-shrink-0 h-100 o_not_editable {not has_desc and 'h-100'}"
-                            t-attf-style="#{forum.image_256 and ('background-image: url(%s);' % website.image_url(forum, 'image_256'))} background-position: center;"/>
+                        <div t-attf-class="oe_img_bg rounded shadow-sm o_wforum_forum_card_bg flex-shrink-0 h-100 o_editable #{not forum.image_1920 and not editable and 'opacity-0'}"
+                        t-field="forum.image_1920" t-options="{'widget': 'image'}" />
                     </a>
                 </div>
-                <div t-attf-class="mt-2 #{has_desc and 'col-md-8 mt-md-0'}">
+                <div class="mt-2 col-md-8 mt-md-0 d-flex flex-column">
                     <a t-attf-href="/forum/#{slug(forum)}" class="text-reset" t-att-title="forum.name">
-                        <h3 t-attf-class="#{has_desc and 'h4' or 'h5 m-0'}" t-field="forum.name"/>
+                        <h3 class="h4" t-field="forum.name"/>
                     </a>
-                    <p t-attf-class="m-0 #{has_desc and not no_description_exist and 'text-muted' or not has_desc and 'css_non_editable_mode_hidden'}"
+                    <p class="m-0 flex-grow-1"
                     placeholder="Description"
-                    t-field="forum.description"/>
+                    t-field="forum.teaser"/>
+                    <div t-if="opt_post_count or opt_last_post" class="row">
```

Comment:
or just do it in extra view with customize show instead.

If view is not active, no request and no extra code !